record_archiver/tests/test_archive.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2015 Guewen Baconnier (Camptocamp SA)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from datetime import datetime, timedelta
+import openerp.tests.common as common
+
+
+class TestArchive(common.TransactionCase):
+
+    def setUp(self):
+        super(TestArchive, self).setUp()
+        Partner = self.env['res.partner']
+        self.partner1 = Partner.create(
```

Comment:
pls rewrite to:
1. use SavepointCase
2. setUp -> setUpClass
3. use tracking_disable on create